guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -102,11 +102,14 @@ endif::[]
 
 .Procedure
 
-. In the {Project} web UI, navigate to *Content* > *Products* and select the product that you want to use, and then click *Create Repository*.
+. In the {Project} web UI, navigate to *Content* > *Products* and select the product that you want to use, and then click *New Repository*.
 . In the *Name* field, enter a name for the repository.
 {ProjectNameX} automatically completes the *Label* field based on what you have entered for *Name*.
 . From the *Type* list, select the type of repository.
 You can select either a repository for RPM files (`yum`), Docker images (`docker`), Files (`file`) or other.
+. If `yum` content type is selected, you can restrict whether the repository is available to a host based on the host's architecture and OS version.
```

Comment:
This isn't a step. Don't use substeps. Just integrate the non-step and the substep into one step.

Reply:
Ack. I've integrated this text into the step above as it's related to Type. And I've changed the substeps to individual/sequential steps.